tools/generate_hotkeys.py

```diff
@@ -1,36 +1,69 @@
 #!/usr/bin/env python3
 from collections import defaultdict
 from pathlib import Path, PurePath
+from typing import Dict, List, Tuple
 
 from zulipterminal.config.keys import HELP_CATEGORIES, KEY_BINDINGS
 
 
-categories = defaultdict(list)
-for item in KEY_BINDINGS.values():
-    categories[item["key_category"]].append((item["help_text"], item["keys"]))
-
+# Path to original docs/hotkeys.md
```

Comment:
Let's try an see if we can simplify the diffs here.
Why not first create the functions in their original place in one commit. Then change the ordering on those functions in another commit. We can squash them later while merging, if needed.

Also, I don't think we need this and the below comment now that they are pretty obvious?

Reply:
Sure, I will remove the comments and split the ordering of functions into 2 commits.

Reply:
Here don't move `OUTPUT_FILE` and `SCRIPT_NAME` then look at the diffs.
You can also move the creation of `various_key_combination()` into another commit. That will make the diffs even more easier to read.